frontend/src/pages/PrivacyPolicy.tsx

```diff
@@ -45,12 +45,10 @@ const PrivacyPolicy = () => {
           How do we store your data? <br />
           Newsletter and communication What are your data protection rights?{" "}
           <br />
-          What are cookies? <br />
-          How do we use cookies? <br />
-          How to manage your cookies? <br />
-          Privacy policies of other websites <br />
-          Changes to our privacy policy <br />
-          How to contact us? <br />
+          What are cookies? <br /> How do we use cookies? How to manage your
+          cookies? <br />
+          Privacy policies of other websites Changes to our privacy policy How
+          to contact us? <br />
           How to contact the appropriate authorities
```

Comment:
I think this can use `<ul></ul>` so it is more readable and also semantically we are listing items